docs/rules/sort-imports.md

```diff
@@ -0,0 +1,167 @@
+# Import Sorting (sort-imports)
+
+When declaring multiple imports, some developers prefer to sort local import names alphabetically to be able to find necessary imports easier at the later time.
```

Comment:
Please show some examples here. This section should explain the concept behind the rule.